nextcloud_install_production.sh

```diff
@@ -704,9 +704,14 @@ rm -f *.zip.*
 done;
 
 # Install virtual kernels
-apt install linux-tools-virtual-hwe-16.04-edge linux-cloud-tools-virtual-hwe-16.04-edge  -y
-apt install linux-image-virtual-hwe-16.04-edge -y
-apt install linux-virtual-hwe-16.04-edge -y
+apt purge -y $(dpkg-query -W -f='${binary:Package}\n' 'linux-image-4*' 'linux-headers-4*' | grep -v 4.4.0-64-generic)
+apt install linux-image-virtual-lts-xenial linux-headers-virtual-lts-xenial -y
```

Comment:
save for later

apt purge -y $(dpkg-query -W -f='${binary:Package}\n' 'linux-image-4*' 'linux-headers-4*' | grep -v 4.4.0-64-generic)

+apt install linux-image-virtual-lts-xenial linux-headers-virtual-lts-xenial -y
 +apt install linux-image-extra-virtual-lts-xenial linux-tools-virtual-lts-xenial -y
 +
 +# Load aufs 
 +apt install aufs-tools -y
 +modprobe aufs
 +echo aufs >> /etc/modules